impl/src/main/java/org/ehcache/config/builders/CacheManagerBuilder.java

```diff
@@ -132,7 +135,7 @@ T cast(EhcacheManager ehcacheManager) {
    * @see CacheConfigurationBuilder
    */
   public <K, V> CacheManagerBuilder<T> withCache(String alias, CacheConfiguration<K, V> configuration) {
-    return new CacheManagerBuilder<>(this, configBuilder.addCache(alias, configuration));
+    return new CacheManagerBuilder<>(this, configBuilder.withCache(alias, configuration));
```

Comment:
This now silently replaces an existing cache.  I think `withCache` implies that but it'll need to be mentioned in a README or release notes for this release.

Reply:
This is likely to turn in to a much bigger conversation involving an internal TLA.

Reply:
Management reports no TLA required